packages/frontend/editor-ui/src/components/MainHeader/WorkflowDetails.vue

```diff
@@ -679,7 +679,11 @@ const onBreadcrumbsItemSelected = (item: PathItem) => {
 					@item-selected="onBreadcrumbsItemSelected"
 				>
 					<template #append>
-						<span v-if="projectsStore.currentProject" :class="$style['path-separator']">/</span>
+						<span
+							v-if="projectsStore.currentProject || projectsStore.personalProject"
```

Comment:
nit: Should we prefer `??` here? Not that it really matters but I think we usually have `typescript-eslint/prefer-nullish-coalescing` rule which doesn't hit here within the vue code.